### PR TITLE
custom lessons created by admin endpoint

### PR DESCRIPTION
**Pull request by AngelaLuo49021**

### CODE REVIEW

File: src/components/ExampleComponent/ExampleComponent.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { withFirebase } from "utils/Firebase";
 const ExampleComponent = ({ firebase }) => {
```
Comment (chyku) [inline src/components/ExampleComponent/ExampleComponent.js:6]: make sure to comment this so we know it's an example LOL mb for not telling you to earlier

File: src/utils/Firebase/firebase.js [hunk 2]
```diff
@@ -26,6 +26,11 @@ class Firebase {
 
     this.googleProvider = new app.auth.GoogleAuthProvider();
   }
+  //Data access APIs
+  customLessons = adminAccountId =>
+    this.db
+      .collection(`custom_lesson`)
+      .where("adminAccountId", "==", adminAccountId);
```
Comment (chyku) [inline src/utils/Firebase/firebase.js]: can u put this underneath w/ the example endpoint

**Review by chyku — CHANGES_REQUESTED** [review]
tiny logistical things mb

**Review by chyku — APPROVED** [review]
SICK

**Review by rxun — APPROVED** [review]

**Review by chyku — APPROVED** [review]

**Pull request merged**